### PR TITLE
quickfix to remove Appsody containers on stop-all command

### PR DESCRIPTION
**Pull request by jcockbain96**
**Problem**
The stop-all command does stop Appsody project containers, as the code only checks for codewind prefixes on the image.
https://github.com/eclipse/codewind/issues/224

**Solution**
Add a check for `appsody` in the images (quickfix), this will also remove existing Appsody projects. A better solution will be required after ifix, one in which we can determine codewind Appsody projects only.

**Tests**
Manual

### CODE REVIEW

**Comment by makandre** [comment]
I don't think we should remove appsody stack images.  Those are like base images that appsody applications use.  Deleting them would be like us deleting the base `node` image that we build our images from

When removing container, could we check for `appsody` image name *and* that the container name starts with `cw-`?  That makes it less likely we remove appsody containers started outside Codewind by mistake.

**Comment by jcockbain96** [comment]
I agree we shouldn't - will add the container name check too.

File: actions/remove.go [hunk 1]
```diff
@@ -22,16 +22,17 @@ import (
 //RemoveCommand to remove all codewind and project images
 func RemoveCommand(c *cli.Context) {
 	tag := c.String("tag")
-	imageArr := [4]string{}
+	imageArr := [5]string{}
 	imageArr[0] = "eclipse/codewind-pfe"
 	imageArr[1] = "eclipse/codewind-performance"
 	imageArr[2] = "eclipse/codewind-initialize"
 	imageArr[3] = "cw-"
+	imageArr[4] = "appsody"
```
Comment (makandre) [inline actions/remove.go]: Is this removing appsody images?

Comment (jcockbain96) [reply actions/remove.go]: It's not removing them, but wasn't supposed to be left in the list. https://github.com/eclipse/codewind-installer/pull/106 removes it.

**Pull request merged**